### PR TITLE
feat(GRO-856): shuffled featured partners carousel

### PR DESCRIPTION
**Pull request by dzucconi**
The type of this PR is: **Feature**

This PR solves [GRO-856]

### Description

Implements a seeded shuffle using the Express request ID as a seed.

[GRO-856]: https://artsyproduct.atlassian.net/browse/GRO-856?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by damassi — APPROVED** [review]
This is rad and very simply done! My only API/DX design question is it would be nice if we could move the `useMemo` from the call-site (`PartnersFeaturedCarousel`) into the hook itself, perhaps by passing the dependency array into the hook as an arg during init. If not no prob.

File: src/v2/Apps/Partners/Components/PartnersFeaturedCarousel.tsx [hunk 1]
```diff
@@ -13,10 +14,11 @@ const PartnersFeaturedCarousel: FC<PartnersFeaturedCarouselProps> = ({
   viewer,
 }) => {
   const profiles = extractNodes(viewer.orderedSet?.orderedItemsConnection)
+  const { shuffled } = useStableShuffle({ array: profiles })
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/v2/Apps/Partners/Components/PartnersFeaturedCarousel.tsx:17]: `items` would work

[the remark this answers is not in the document]
Comment (dzucconi) [reply src/v2/Apps/Partners/Components/PartnersFeaturedCarousel.tsx:17]: Loaded? It's a shuffle function. Got any ideas?

[the remark this answers is not in the document]
Comment (damassi) [reply src/v2/Apps/Partners/Components/PartnersFeaturedCarousel.tsx:17]: `array` -> `items`

[the remark this answers is not in the document]
Comment (dzucconi) [reply src/v2/Apps/Partners/Components/PartnersFeaturedCarousel.tsx]: ✔️

**Pull request merged**